### PR TITLE
Ruff edit(read details)

### PR DESCRIPTION
**Pull request by EternalMC**
I don't speak Chinese but I do speak English, so the edits are a little ruff(needed fixing)
Google translate save lives. :)
I'm also willing to edit the rest.

### CODE REVIEW

File: 02-install/02-01-install_rust_on_linux.md [hunk 1]
```diff
@@ -1,50 +1,52 @@
...
 `curl -sSf https://static.rust-lang.org/rustup.sh | sh`
 
-打开终端执行如上命令即可。
 
+Open the terminal and execute the above command.
 
-**注意**
 
-除了稳定版之外，Rust 还提供了 Beta 和 Nightly 版本，下载地址如下：
+**note**
```
Comment (wayslog) [inline 02-install/02-01-install_rust_on_linux.md]: maybe Notice or attention ?

File: 02-install/02-01-install_rust_on_linux.md [hunk 2]
```diff
@@ -1,50 +1,52 @@
...
 
 `uname -a`
 
-结果如下图所示：
+The result is shown below：
 
 ![check system info](../image/01-install-on-linux-check-system.png)
 
-如上图所示，如果是 **x86_64** 则证明是64位系统，需要[下载](https://static.rust-lang.org/dist/rust-1.5.0-x86_64-unknown-linux-gnu.tar.gz)64位安装包；
+As shown above, if it is **x86_64** 64-bit system，to [download](https://static.rust-lang.org/dist/rust-1.5.0-x86_64-unknown-linux-gnu.tar.gz)64-bit installation package；
```
Comment (wayslog) [inline 02-install/02-01-install_rust_on_linux.md]: maybe the word must have space charactor between `download` and `64-bit` ?

File: 02-install/02-01-install_rust_on_linux.md [hunk 3]
```diff
@@ -1,50 +1,52 @@
...
+Rust two installation methods for Linux users：
 
-### 1、直接下载安装包：
+### 1、Download the installation package directly：
 
-直接下载安装包的话需要检查一下你当前操作系统是64位还是32位，分别下载对应的安装包。
+To download the installation package directly, you need to check whether your current operating system is 64-bit or 32-bit, and download the corresponding installation package.
 
-查看操作系统请在终端执行如下命令:
+To view the operating system, execute the following command in the terminal:
```
Comment (wayslog) [inline 02-install/02-01-install_rust_on_linux.md]: specially `uname` command was only works well in linux/macOS .

File: 02-install/02-01-install_rust_on_linux.md [hunk 4]
```diff
@@ -1,50 +1,52 @@
...
+
+
+In addition to the stable version，rust also provided Beta with Nightly version，download address is as follows：
 https://www.rust-lang.org/downloads.html
 
-如果你不想安装 Rust 在你的电脑上，但是你还是像尝试一下 rust，那么这里有一个在线的环境：http://play.rust-lang.org/
+If you don't want to install rust on your computer，but you still want try rust，Then here is an online environment：http://play.rust-lang.org/
 
-中国科学技术大学镜像源包含 [rust-static](http://mirrors.ustc.edu.cn/rust-static/)，梯子暂时出问题的同学可以尝试从这里下载编译器；除此之外。还有 Crates 源，详见[这里的说明](https://servers.ustclug.org/2016/01/mirrors-add-rust-crates/)。
+University of Science and Technology of China mirror source contains [rust-static](http://mirrors.ustc.edu.cn/rust-static/)，Students who have problems with the ladder temporarily can try to download the compiler from here; and also Crates source，See details[Instructions here](https://servers.ustclug.org/2016/01/mirrors-add-rust-crates/)。
```
Comment (wayslog) [inline 02-install/02-01-install_rust_on_linux.md]: This is Chinese mainland's mirror, I think that some one not live in there may never use that. You could delete permanently.

**Comment by wayslog** [comment]
sorry for rely later.

File: 02-install/02-01-install_rust_on_linux.md (hunk 1 above)
Comment (EternalMC) [reply 02-install/02-01-install_rust_on_linux.md]: edit done

File: 02-install/02-01-install_rust_on_linux.md (hunk 3 above)
Comment (EternalMC) [reply 02-install/02-01-install_rust_on_linux.md]: changed

File: 02-install/02-01-install_rust_on_linux.md (hunk 2 above)
Comment (EternalMC) [reply 02-install/02-01-install_rust_on_linux.md]: done

**Comment by EternalMC** [comment]
Everything has been resolved.

**Comment by EternalMC** [comment]
Also 
https://uhrp.org/sites/default/files/styles/large/public/field/image/tibet-protest.JPG?itok=K3545exA

**Comment by EternalMC** [comment]
@wayslog

**Pull request closed without merging**